docs/event/metadata.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 2
+---
+
+# Metadata
+
+(🟦) Metadata fields marked with this symbol are *Opencast-managed*: they are read-only for users/external applications. All other fields can be freely changed, as long as validity checks pass.
+
+#### General
+- `id: ID` 🟦: unique among all events.
```

Comment:
As discussed recently, this needs clearer definition. Is the `id` unique to the system, tenancy ( or even globally)? I think the feeling was that `id+organization` should be unique because of the possibility of loading the same MediaPackage into different tenancies in the same system.

Need to add `organization: NonBlankString` ?

Reply:
We discussed this in a meeting and concluded that we want to deal with multi tenancy on a higher level. I added a section about this in https://github.com/opencast/data-model/commit/f05a7cf7f8177f1ffd21bc75516b0e60be4f323b